### PR TITLE
Adopt the OpenStreetMap US Code of Conduct

### PR DESCRIPTION
**Pull request by ZeLonewolf**
This proposes to adopt the OSM US Code of Conduct.

This PR is initially draft so that we can work out the language.  Please hold off on adding your approval until this PR is marked ready for review.

Adoption procedure prior to merge:
* [ ] All maintainers concur with adopting this CoC, as we cannot have maintainers which do not agree to uphold commen standards of conduct
  * [x] Brian
  * [x] Minh
  * [x] Adam
  * [x] Ariel
  * [x] Clay
  * [x] Josh
  * [ ] Martin
  * [ ] Drew
  * [x] Zeke
* [x] OSM US CoC Committee concurs with the text of this file
* [x] License issues have been dealt with

Note that any maintainers who wish to withdraw their maintainership for any reason may choose to do so at any time and for any reason without prejudice against returning to the maintainer team at some future date.  I intend that the addition of new maintainers in the future will be contingent on the new maintainer adding their approval to this PR, signaling concurrence with the CoC.

### CODE REVIEW

**Comment by adamfranco** [comment]
I read through the `CODE_OF_CONDUCT.md` and re-read the linked resources. Nothing jumped out at me as problematic and I feel that this CoC is appropriate for this project. Overall, I particularly like this CoC's identification of "Encouraged Behaviors" in addition to those prohibited/"Untolerated". Avoiding the prohibited behaviors is a really low bar of acceptable behavior, the positive behaviors are things that individuals potentially in conflict can look to as better ways of interacting. I support adoption of this CoC.

File: CODE_OF_CONDUCT.md
Comment (adamfranco) [inline CODE_OF_CONDUCT.md:58]: The OSM-US CoC looks to be licensed under either the [Creative Commons Attribution-ShareAlike 3.0](http://creativecommons.org/licenses/by-sa/3.0/) or [Creative Commons Attribution-ShareAlike 2.0](https://wiki.openstreetmap.org/wiki/Wiki_content_license).

As the `CODE_OF_CONDUCT.md` is not executable code that must be linked or compiled into any software package, my understanding is that it would be fine to include this file in the Americana repository under its existing Creative Commons Attribution-ShareAlike license without any public-domain waver or copyright release. My lay understanding of the applicable law is that such a license on this one file (with appropriate in-file links) would be sufficient to fulfill the attribution and share-alike demands of the license and would not force any of the rest of the code or other files also included in this repository to have the same license.

Additionally, the OSM-US is likely not in a position to grant such a release as the OSM-US CoC itself drew from upstream Creative Commons Attribution-ShareAlike licensed sources.

Comment (ZeLonewolf) [reply CODE_OF_CONDUCT.md:58]: Copyright is not just limited to executable code - it applies to any creative work.  We can't declare this repository to be CC0/public domain if there are portions of the repository that have copyright requirements (specifically - attribution).  If we (OSM US) aren't able to produce PD-compatible CoC text, I don't see how we (Americana project) can include the content in the repo.

Comment (ZeLonewolf) [reply CODE_OF_CONDUCT.md:58]: Per Slack discussion, I've updated the README with an exception disclaimer and we'll include the attribution piece here in this file.

Comment (adamfranco) [reply CODE_OF_CONDUCT.md:58]: Correct that copyright isn't limited to executable code. That said, the "viral" nature of share-alike licenses (e.g. GPL) has been determined in litigation to stop at the compiling/linking/single-executable boundary and is separate from "collections of differently licensed applications".

In my mind, this file and the code are separate and simply "collected" together rather than being a single "work".

**Review by zekefarwell — APPROVED** [review]
I approve of adopting this code of conduct.  

I have a slight issue with the wording of this sentence in the untolerated behaviors list:
> Conduct or speech which might be considered sexist, racist, homophobic, transphobic, xenophobic, ableist or otherwise discriminatory or offensive in nature;

The phrasing "might be considered" makes this quite a vague statement and opens the door to an unreasonable person claiming that a behavior was discriminatory even though nobody else sees it as such.  This sentence is clearer and more specific:

> Conduct or speech that is sexist, racist, homophobic, transphobic, xenophobic, ableist or otherwise discriminatory or offensive in nature;

 What is the proper channel for suggesting this minor wording change?

**Comment by ZeLonewolf** [comment]
I've updated this PR to keep things simple: OSM Americana adopts the OSM US Code of Conduct unless later revoked by the maintainer team.

**Review by akadouri — APPROVED** [review]
Not sure three commits a long time ago still makes me a maintainer but looks good to me.

**Review by claysmalley — APPROVED** [review]

**Review by 1ec5 — APPROVED** [review]

**Review by adamfranco — APPROVED** [review]

**Review by jleedev — APPROVED** [review]

**Review by zekefarwell — APPROVED** [review]

**Comment by ZeLonewolf** [comment]
Given the lack of opposition thus far, I'm moving this forward with the maintainers that have added their approvals thus far.  @Pengor, @Mashin6, please add your approvals if you intend to continue as repository maintainers.

**Review by Pengor — COMMENTED** [review]
I approve!

**Comment by wmisener** [comment]
I approve!

**Review by wmisener — COMMENTED** [review]
I approve!

**Pull request merged**
